v1-preparations/LIST_OF_PREVIEW_FEATURES_FOR_V1.md

```diff
@@ -14,7 +14,8 @@
 * [snowflake_external_function](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/external_function)
 * [snowflake_external_functions](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/external_functions)
 * [snowflake_external_table](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/external_table)
-* [snowflake_external_tables](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/external_tables)
+* [snowflake_external_tables](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/data-sources/external_tables) (datasource)
+* [snowflake_external_volume](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/external_volume)
 * [snowflake_failover_group](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/failover_group)
 * [snowflake_failover_groups](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/failover_groups)
```

Comment:
I think that failover groups should also be fixed. Check other resources as well.

Reply:
Oky, I will check with the next PR

Reply:
Done in https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3183